lua/README.md

```diff
@@ -81,8 +84,8 @@ The first thing to do when writing a lua script for MQ2 is to `require('mq')`. T
 `print('string')`
 :   as noted before
 
-`mq.delay(val, callback)`
-:   where val can be an integer (which denotes milliseconds of delay) or a string that ends in 's' 'm' or 'ms' to have delays with human readable durations and callback is an optional lua function which evaluates to true or false to decide whether to end the delay early
+`mq.delay(val, --[[optional]]callback)`
+:   where val can be an integer (which denotes milliseconds of delay) or a string that ends in 's' 'm' or 'ms' to have delays with human readable durations. The callback is optinal and is a function call condition that returns a bool that can end the delay early.
```

Comment:
"optional" but I also prefer the wording "which evaluates to true or false" over "call condition"